data-model-layer/data-model.md

```diff
@@ -85,9 +85,42 @@ Boolean is a scalar kind.  Its cardinality is two -- either the value 'true' or
 
 #### Integer kind
 
+Integer is a scalar kind and refers to whole numbers without a fractional
+component.
+
+It is important to consider codec and language limitations that may be imposed
+on the serialization of integers from the IPLD Data Model. For example:
+
+* Some codecs, such as DAG-CBOR, will assume that integers must be within the
+  64-bit signed range and reject anything larger.
+* IPLD libraries, such as go-ipld-prime, limit their in-memory representation
+  of the integer kind to the signed 64-bit range.
+* JavaScript has difficulties safely handling and representing integers outside
+  of the 53-bit unsigned range and differentiating between integers and floats
+	where there is no fractional component.
 
 #### Float kind
 
+Float is a scalar kind and refers to a number with a fractional component,
+represented with a decimal point. This may also include numbers where the
+fractional component is zero (although the distinction between these numbers
+and integers is blurred in some cases—specifically the JavaScript memory model).
```

Comment:
maybe spaces around the dash? otherwise it almost reads like a single word. but I'm not an expert at technical writing :)

Reply:
in the code font on here it looks like an en-dash but it's an em-dash, which is longer and shouldn't have spaces around it when used in sentences like this. If you look at it in preview it comes out properly: https://github.com/ipld/specs/blob/rvagg/ieee754-specials/data-model-layer/data-model.md

Reply:
gotcha, sounds good. I guess in here it's monospace.